camera_info_manager/src/camera_info_manager.cpp

```diff
@@ -78,20 +78,29 @@ const std::string
  * @param url default Uniform Resource Locator for loading and saving data.
  */
 CameraInfoManager::CameraInfoManager(
-  rclcpp::Node * node,
-  const std::string & cname,
+  rclcpp::Node * node, const std::string & cname,
   const std::string & url)
-: logger_(node->get_logger()),
+: CameraInfoManager(node->get_node_base_interface(),
+    node->get_node_services_interface(), node->get_node_logging_interface(), cname, url)
+{
+}
+
```

Comment:
Could you also add an overloaded constructor that takes in the `rclcpp::LifecycleNode*` that calls the constructor in L88?